Makefile

```diff
@@ -4,7 +4,7 @@ python_version = 3
 LC = LC_MESSAGES
 
 version := $(shell \
-    python3 -c 'from kiwi.version import __version__; print(__version__)'\
+    python -c 'from kiwi.version import __version__; print(__version__)'\
```

Comment:
Please use a variable for the python program so that it can be easily substituted at `make` invocation.

See https://github.com/livecd-tools/livecd-tools/blob/master/Makefile for an example of how I do it.